decidim-core/app/packs/src/decidim/editor/clipboard_utilities.js

```diff
@@ -0,0 +1,119 @@
+import { BlockEmbed } from "quill/blots/block";
```

Comment:
As you've mentioned that some (or all) of these functions are copied from Quill codebase, I think we should add somekind of header like we've done [other times](https://github.com/decidim/decidim/blob/db5df0171e0d4a5e060248036aed62696b54ec74/decidim-core/app/packs/src/decidim/map/svg-icon.js#L3) that we've needed to work locally with code made by other people. 

I was thinking on something like: 
```javascript
//
// Quill clipboard utilities 
//
// Copyright (c) 2017, Slab
// Copyright (c) 2014, Jason Chen
// Copyright (c) 2013, salesforce.com
// BSD 3-Clause "New" or "Revised" License
//
// Originally copied from https://github.com/quilljs/quill
// Converted from TypeScript to JavaScript
//
```

Feel free to improve it of course

Reply:
Added the notes to the override file instead, see f6f7e96a7f1e4a580db6647461863d12f0816bf7.